frontend/src/hooks/User/activityLogs.js

```diff
@@ -0,0 +1,20 @@
+import useSWR from "swr";
+
+import axios from "../../lib/axios";
+
+export const useActivityLogs = () => {
+  const fetcher = (...args) => axios(...args).then((res) => res.data);
+  const options = {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+    revalidateOnReconnect: true,
+    revalidateOnMount: true,
+  };
+
+  const { data, error } = useSWR(`/api/user/activity-logs`, fetcher, options);
```

Comment:
instead of ` `` ` use `""` since it is just a string
`"/api/user/activity-logs"`